pkg/plugin/manifest/manifest.go

```diff
@@ -60,21 +60,30 @@ func (s *SonobuoyConfig) DeepCopy() *SonobuoyConfig {
 
 // Manifest is the high-level manifest for a plugin
 type Manifest struct {
-	SonobuoyConfig SonobuoyConfig `json:"sonobuoy-config"`
-	Spec           Container      `json:"spec"`
-	ExtraVolumes   []Volume       `json:"extra-volumes,omitempty"`
-	PodSpec        *PodSpec       `json:"podSpec,omitempty"`
+	SonobuoyConfig SonobuoyConfig    `json:"sonobuoy-config"`
+	Spec           Container         `json:"spec"`
+	ExtraVolumes   []Volume          `json:"extra-volumes,omitempty"`
+	PodSpec        *PodSpec          `json:"podSpec,omitempty"`
+	Configs        map[string]string `json:"configs,omitempty"`
```

Comment:
Ensure I'm consistent about config/configs; unless another word entirely works better. Does get confusing because of the `sonobuoyConfig`

Reply:
Using ConfigMap instead; I think thats more clear even if not completely accurate